src/components/AccountCreation/AccountCreation.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+
+// Define the type for the prop
+interface AccountCreationProps {
+  onAccountCreate: (username: string, email: string, password: string) => void;
+}
+
+const AccountCreation: React.FC<AccountCreationProps> = ({
+  onAccountCreate,
+}) => {
+  const [username, setUsername] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleCreateAccount = async () => {
+    // Basic validation: Check if both username, email and password are not empty after trimming whitespace
+    if (username.trim() && email.trim() && password.trim()) {
+      // If username, email and password are not empty, proceed with creating the account
+
+      // Call the parent component's callback with the entered username and password
+      onAccountCreate(username, email, password);
+
+      try {
+        // Communicate with the backend team to determine the format of the request
+        const response = await fetch("/users", {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            username,
+            email,
+            password,
+          }),
+        });
+
+        if (response.ok) {
+          // Account creation successful, handle accordingly (e.g., show a success message)
+          console.log("Account created successfully");
+        } else {
+          // Handle account creation failure (e.g., show an error message)
+          console.error("Account creation failed.");
+        }
+      } catch (error) {
+        console.error("Error during account creation:", error);
+      }
+    } else {
+      // If either username, email or password is empty, show an alert
+      // Handle validation error by showing a simple pop-up alert message
+      alert("Please enter a valid username, email and password.");
+    }
+  };
+
+  return (
+    <div>
```

Comment:
My chrome console is pointing out that password inputs should always be wrapped in a `form` element:

<img width="570" alt="Screen Shot 2023-11-29 at 5 19 34 PM" src="https://github.com/Code-the-Dream-School/ee-prac-team2-front/assets/12771651/4fb65995-061c-44d2-8e4c-94c3949dec39">

If you replace this `div` with a form`, you can remove the event handler that's currently the `onClick` for the submit button, and move it to the form's `onSubmit` event handler: https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event